buildNumber missing when building downloaded .zip

OSX 10.13.4
Java 1.8.0_144
Maven 3.5.3

Error on Mvn Cln package

Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.864 s <<< FAILURE! - in org.phoenicis.javafx.AppConfigurationInitialisationTest
testAppConfigurationInitialisation(org.phoenicis.javafx.AppConfigurationInitialisationTest)  Time elapsed: 0.83 s  <<< ERROR!
java.lang.RuntimeException: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'org.phoenicis.javafx.controller.ControllerConfiguration': Unsatisfied dependency expressed through field 'viewsConfiguration'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'org.phoenicis.javafx.views.ViewsConfiguration': Injection of autowired dependencies failed; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'buildNumber' in value "${buildNumber}"
	at org.phoenicis.javafx.AppConfigurationInitialisationTest.testAppConfigurationInitialisation(AppConfigurationInitialisationTest.java:22)
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'org.phoenicis.javafx.controller.ControllerConfiguration': Unsatisfied dependency expressed through field 'viewsConfiguration'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'org.phoenicis.javafx.views.ViewsConfiguration': Injection of autowired dependencies failed; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'buildNumber' in value "${buildNumber}"
	at org.phoenicis.javafx.AppConfigurationInitialisationTest.lambda$testAppConfigurationInitialisation$0(AppConfigurationInitialisationTest.java:22)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'org.phoenicis.javafx.views.ViewsConfiguration': Injection of autowired dependencies failed; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'buildNumber' in value "${buildNumber}"
	at org.phoenicis.javafx.AppConfigurationInitialisationTest.lambda$testAppConfigurationInitialisation$0(AppConfigurationInitialisationTest.java:22)
Caused by: java.lang.IllegalArgumentException: Could not resolve placeholder 'buildNumber' in value "${buildNumber}"
	at org.phoenicis.javafx.AppConfigurationInitialisationTest.lambda$testAppConfigurationInitialisation$0(AppConfigurationInitialisationTest.java:22)


[Build_output.txt](https://github.com/PhoenicisOrg/phoenicis/files/2032909/Build_output.txt)

--- a/phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/ViewsConfiguration.java
+++ b/phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/ViewsConfiguration.java
@@ -43,13 +43,13 @@ public class ViewsConfiguration {
     @Value("${application.name}")
     private String applicationName;
 
-    @Value("${application.version}")
+    @Value("${application.version}:unknown")
     private String applicationVersion;
 
-    @Value("${application.gitRevision}")
+    @Value("${application.gitRevision}:unknown")
     private String applicationGitRevision;
 
-    @Value("${application.buildTimestamp}")
+    @Value("${application.buildTimestamp}:unknown")
     private String applicationBuildTimestamp;
 
     @Value("${application.user.engines}")

--- a/pom.xml
+++ b/pom.xml
@@ -164,6 +164,7 @@
                     </execution>
                 </executions>
                 <configuration>
+                    <revisionOnScmFailure>unknown</revisionOnScmFailure>
                     <getRevisionOnlyOnce>true</getRevisionOnlyOnce>
                     <doCheck>false</doCheck>
                     <doUpdate>false</doUpdate>

